Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -493,10 +493,11 @@ extension ProjectDescription.DeploymentTarget {
         package: [PackageInfo.Platform],
         packageName _: String
     ) throws -> Self {
-        if let packagePlatform = package.first(where: { $0.platformName == platform.rawValue }) {
+        if let packagePlatform = package.first(where: { $0.tuistPlatformName == platform.rawValue }) {
             switch platform {
             case .iOS:
-                return .iOS(targetVersion: packagePlatform.version, devices: [.iphone, .ipad])
+              let hasMacCatalyst = package.contains(where: { $0.platformName == "maccatalyst" })
+              return .iOS(targetVersion: packagePlatform.version, devices: hasMacCatalyst ? [.iphone, .ipad, .mac] : [.iphone, .ipad])
```

Comment:
is adding `.mac` enough to support maccatalyst?

Reply:
AFAIK that's exactly what mac is supposed to do

Reply:
Cool! Never really used maccatylyst, so was not completely sure 😅